src/views/@common/hooks/useGetCheckApplication.ts

```diff
@@ -0,0 +1,24 @@
+import { isAxiosError } from 'axios';
+import { useEffect, useState } from 'react';
+
+import api from './api';
+
+const useGetCheckApplication = () => {
+  const [status, setStatus] = useState(0);
+  const getCheckApplication = async () => {
+    try {
+      const response = await api.get('/application/check');
+      setStatus(response.status);
+    } catch (err) {
+      isAxiosError(err) && setStatus(err.response?.status || 0);
+    }
+  };
+
+  useEffect(() => {
+    getCheckApplication();
+  }, []);
+
+  return status;
```

Comment:
```suggestion
  return getCheckApplication;
```
status나 응답 데이터를 반환해주는 친구는 여기 리턴문이 아닌 getCheckApplication 함수 내의 리턴문으로 넣어줘야할 것 같고, 여기서는 StartButton을 클릭했을 때 실행시킬 통신 함수를 보내줍시다!